docs/connectors/index.mdx

```diff
@@ -1,11 +1,170 @@
-# Data Sources and Sinks
+# Connectors: connect to data sources and sinks
 
 A Feldera pipeline can process data from multiple heterogeneous sources and
-produce outputs to multiple heterogeneous destinations.  To this end it relies
-on a growing library of input and output connectors.
+produce outputs to multiple heterogeneous destinations. To this end it relies
+on a growing library of **[input](sources)** and **[output](sinks)** connectors.
 
-```mdx-code-block
-import DocCardList from '@theme/DocCardList';
+:::caution
 
-<DocCardList />
+The connectors API is still evolving and might see backwards incompatible changes.
+
+:::
+
+## Basics
+
+The connector object is the abstraction used to define data coming in from data sources
```

Comment:
A connector is a Feldera API object that describes an external data source or sink such as a database table or a Kafka topic.  Connectors work together with other API objects, namely programs, and pipelines, to build end-to-end streaming analytics pipelines.